client/src/components/components.test.js

```diff
@@ -90,7 +90,7 @@ const setCurrentPicIndex = setCurrentPicIndexSpy;
 
 describe('<App />', () => {
   it('should render a PhotoContainer component', () => {
-    const wrapper = shallow(<App />);
+    const wrapper = shallow(<App siteId={5}/>);
```

Comment:
did this help with the test?

Reply:
Sadly it didn't.  That was an attempt that didn't work that I left in accidentally... oops!  There's a whole thing you have to do with Enzyme and RR that I'm going to look into tomorrow.